### PR TITLE
Block SFVC context for Venmo

### PR DESCRIPTION
**Pull request by mnicpt**
Don't show Venmo option when falling back to web because of being inside the context of Safari View Controller.

### CODE REVIEW

File: src/funding/venmo/config.jsx [hunk 1]
```diff
@@ -46,6 +47,10 @@ export function getVenmoConfig() : FundingSourceConfig {
             [ BUTTON_COLOR.GOLD ]:   BUTTON_COLOR.BLUE,
             [ BUTTON_COLOR.BLUE ]:   BUTTON_COLOR.SILVER,
             [ BUTTON_COLOR.SILVER ]: BUTTON_COLOR.BLUE
+        },
+
+        eligible: () => {
```
Comment (gregjopa) [inline src/funding/venmo/config.jsx:52]: Could we incorporate this check in the `supportedNativeBrowser` prop? 

- https://github.com/paypal/paypal-checkout-components/blob/master/src/zoid/buttons/component.jsx#L506-L510
- https://github.com/paypal/paypal-checkout-components/blob/master/src/zoid/buttons/util.js#L22-L40

Venmo is already using the `supportedNativeBrowser` prop to help determine eligibility.

Comment (mnicpt) [reply src/funding/venmo/config.jsx:52]: @gregjopa This is a great point!  Taking a look at it.

File: package.json [hunk 2]
```diff
@@ -29,7 +29,7 @@
         "jest-ssr": "jest test/ssr --env=node --no-cache",
         "jest-e2e": "rm -f ./test/e2e/screenshots/*.png && jest test/e2e",
         "get-size": "gzip -9 < dist/size.min.js | wc -c",
-        "check-size": "npm run webpack-size && if [ $(npm run -s get-size) -ge 70000 ]; then echo 'Bundle size greater than 65k'; exit 1; else npm run delete-size; fi;",
+        "check-size": "npm run webpack-size && if [ $(npm run -s get-size) -ge 85000 ]; then echo 'Bundle size greater than 65k'; exit 1; else npm run delete-size; fi;",
```
Comment (bluepnume) [inline package.json:32]: This is concerning. The script size jumped 15k?

File: src/zoid/buttons/util.js [hunk 3]
```diff
@@ -28,6 +28,10 @@ export function isSupportedNativeBrowser() : boolean {
         return false;
     }
 
+    if (isSFVC()) {
```
Comment (bluepnume) [inline src/zoid/buttons/util.js:31]: LGTM!

File: package.json (hunk 2 above)
Comment (gregjopa) [reply package.json:32]: I tested this change locally and ran `npm run check-size` and it seemed to pass fine without changing this threshold. I got the following output:
```
Final bundle size: 66362
```

**Review by gregjopa — APPROVED** [review]
The change looks good! I left a comment about backing out the `check-size` change since it shouldn't be needed.

**Comment by mnicpt** [comment]
@gregjopa So odd. Mine showed 75k. Thought that seemed high.  Will put back to 70k.

**Comment by mstuart** [comment]
:+1: lgtm

**Pull request merged**